docs/README.md

```diff
@@ -12,14 +12,23 @@
 [![npm version](https://badge.fury.io/js/xstate.svg)](https://badge.fury.io/js/xstate)
 <img src="https://opencollective.com/xstate/tiers/backer/badge.svg?label=sponsors&color=brightgreen" />
 
-JavaScript and TypeScript [finite state machines](https://en.wikipedia.org/wiki/Finite-state_machine) and [statecharts](https://www.sciencedirect.com/science/article/pii/0167642387900359/pdf) for the modern web.
+Handle complexity anywhere JavaScript runs using statecharts.
```

Comment:
Could this be “Handle complex states anywhere JavaScript runs using statecharts.”? Just to be a bit more specific about the complexity, and as an intro for the following list?

Reply:
I would rephrase "anywhere JavaScript runs" to "in JavaScript" as that reads a bit more naturally